ublox/src/ubx_packets/packets.rs

```diff
@@ -1986,6 +1986,50 @@ struct EsfMeas {
 //     msss: u32,
 // }
 
+#[ubx_packet_recv]
+#[ubx(class = 0x28, id = 0x00, fixed_payload_len = 72)]
+struct HnrPvt {
+    i_tow: u32,
+    year: u16,
+    month: u8,
+    day: u8,
+    hour: u8,
+    min: u8,
+    sec: u8,
+    valid: u8,
```

Comment:
I think this should also be a bitfield with values `FullyResolved`/`ValidTime`/`ValidDate`